### PR TITLE
QA: Add travis file for testing

### LINKED ISSUE #12: Setup Travis CI to run tests for BIT React frontend
## Description
As a developer,
I need run tests on every PR on CI,
so that I can immediately get feedback on a build

## Out of Scope
Deploying to a higher environment when the push is on the develop branch

## Acceptance Criteria
- [ ] Build gets triggered on push
- [ ] Tests run for build

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentors.

## Estimation
1-2 hours

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description

Added travis file 

Fixes #12 and replaces PR #22 

### Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
pushing to local fork and travis resulted in passed build

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by mtreacy002** [comment]
Update @anitab-org/bridgeintech-maintainers . This PR is now ready to be (re)-review. Thanks cc @foongminwong and @meenakshi-dhanani

**Review by meenakshi-dhanani — APPROVED** [review]
Cleanup looks good

**Comment by meenakshi-dhanani** [comment]
@anitab-org/bridgeintech-maintainers can someone approve? We've been waiting on a pending approval for a while now

**Review by foongminwong — APPROVED** [review]

**Pull request merged**
